rt/src/hand_off.rs

```diff
@@ -0,0 +1,51 @@
+// Licensed under the Apache-2.0 license.
+
+const FHT_MAGIC: u32 = 0x54484643;
+#[repr(C)]
+#[derive(Clone, Debug, Default)]
+pub struct HandoffParams {
```

Comment:
call it FirmwareHandoffTable, stay consistent with spec

Same for all the fields, keep the naming in line with what is in the spec so there is no confusion. Its fine that you changed from camel case to snake case but don't deviate on the actual text

If there is a reason to change a name, it should be done here and in the spec simultaneously, as part of the same PR

Reply:
This data structure has been moved to the common crate.